payments_config/__init__.py

```diff
@@ -62,6 +64,10 @@ def __init__(self, seller, config):
             raise ValueError('Recurrence should be not set or monthly: {}'.
                              format(self.recurrence))
 
+        if self.user_identification not in self.valid_user_identifications:
+            raise ValueError('user_identification must be one of: {}'.
+                             format(str(self.valid_user_identifications)))
```

Comment:
`str()` is not needed because it happens automatically when the string is formatted. One nice thing you could do is comma separate it: `'...{}'.format(', '.join(self.valid_user_identifications))`